### ISSUE TITLE
Simulator crashing on opening ECAM Status page

### ISSUE THREAD

**Issue by Claudio140** [headwind-msfs/a330-900#30]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2022-04-28T21:25:02+00:00",
  "ref": "v0.106",
  "sha": "bdb813be7ad5cdab27cfc8ef010f05efe6ff80a5",
  "actor": "Revyn112",
  "event_name": "manual"
}
```

### Describe the bug

When performing the pre-flight cockpit preparation, the Simulator crashes with a memory error (segfault?) when opening the ECAM status page.
General aircraft status: APU started, Ext PWR disconnected, IRS still aligning, WX radar switched on, Forward pax door open (via FlyPad).

### Expected behavior

The ECAM status page opens on the lower ECAM display when pressing the corresponding button and shows "Normal".

### Steps to reproduce

1. Load up a flight with the MSFS Flight Planner.
2. Start the aircraft as usual, i.e. BAT/Ext PWR -> Fire test, fuel pumps -> APU + Ext PWR disc -> Light switches (as req.), IRS align, Cond. knobs set.
3. Manually cycle through ECAM pages.
4. Every page works except for the status page when selected last. I don't know if it's the same when selecting the status page earlier though.

### Additional info (optional)

I already had the issue once with I think what was v0.104, but definitely before installing SU9.

### Discord Username (optional)

Claudio140#3237

**Comment by Revyn112** [comment]
Did you perform a clean install (also removing the headwind files in appdata)?

**Comment by Claudio140** [comment]
Apparently I haven't since I just now figured the old versions are in a subfolder (which is also a symlink) called "LocalState/packages" of the folder you guys provided in the release notes. I'm on the MS Store version.

I removed the regular folder from the community folder location because I thought it would be enough. After removing the stuff in AppData, at least for now it works.
Sorry for not following the instructions thoroughly 😄
